languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -9779,6 +9779,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+    <rulegroup id="RISQUE_ET_CHANCE" name="risque et chance">
+        <rule>
+            <pattern>
+                <marker>
+                    <token postag="V (ind|sub).*" postag_regexp="yes" inflected="yes">risquer</token>
+                </marker>
+                <token regexp="yes">de</token>
+                <token regexp="yes">gagner</token>
+            </pattern>
+            <message>Faute de style possible car cette expression peut paraître péjorative.</message>
+            <suggestion><match no="1" postag="(V )(ind|sub)(.*)" postag_regexp="yes" postag_replace="(V avoir ind$3)|(V avoir sub$3)">avoir</match> des chances</suggestion>
+            <example correction="a des chances|ait des chances">Il <marker>risque</marker> de gagner.</example>
+        </rule>
+      <rule>
+          <pattern>
+              <marker>
+                  <token inflected="yes">avoir</token>
+                  <token regexp="yes">des</token>
+                  <token regexp="yes">chances</token>
+              </marker>
+              <token regexp="yes">de|d'</token>
+              <token regexp="yes">perdre|échouer</token>
+          </pattern>
+          <message>Une faute de style est possible car cette expression peut paraître péjorative.
+              Nous vous suggérons  d’employer <suggestion>risquer</suggestion>.</message>
+          <example correction="risquer"><marker>avoir des chances</marker> de perdre</example>
+          <example correction="risquer"><marker>avoir des chances</marker> d'échouer</example>
```

Comment:
- Pas d'alinéa dans le <message>
- Transfer the verb form from avoir to risquer